test/format/ops.c

```diff
@@ -1163,6 +1163,16 @@ ops(void *arg)
                     tinfo->last += range;
                     if (tinfo->last > max_rows)
                         tinfo->last = 0;
+                    /*
+                     * Edge case: If we are truncating to the end of the table and we're doing
+                     * mirroring with both a VLCS and FLCS table, then we can get a false mismatch
+                     * failure if the truncate happens before the first append-only insert to the
+                     * VLCS table. So if we have VLCS and FLCS mirrors, set last to the second last
+                     * key.
```

Comment:
I found the explanation of this edge case on the ticket easier to understand than the comment here. So maybe reword or copy the other text?

I think the difference is that the ticket spells out the problematic sequence of events with more detail.

Reply:
I pulled that in an got rid of the specific key numbers. Please reread.